examples/client.py

```
@@ -26,6 +26,36 @@
+#   export REQUESTS_CA_BUNDLE=/path/to/mitmproxy-ca.pem
+# setup proxy to elabapi client's config
+proxy_url = os.getenv("HTTPS_PROXY") or os.getenv("HTTP_PROXY")
+if proxy_url:
+    # make sure underlying libraries also see the proxy (requests / subprocesses)
+    os.environ.setdefault("HTTPS_PROXY", proxy_url)
+    os.environ.setdefault("HTTP_PROXY", proxy_url)
+    try:
+        configuration.proxy = proxy_url
+    except Exception:
+        # some generated clients don't expose 'proxy', so ignore safely
+        pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)